src/NuGet.Server.V2/NuGetV2WebApiEnabler.cs

```diff
@@ -66,11 +67,14 @@ internal static IEdmModel BuildNuGetODataModel()
         {
             var builder = new ODataConventionModelBuilder();
 
+            builder.DataServiceVersion = new Version(2, 0);
+            builder.MaxDataServiceVersion = builder.DataServiceVersion;
+
             var packagesCollection = builder.EntitySet<ODataPackage>("Packages");
             packagesCollection.EntityType.HasKey(pkg => pkg.Id);
             packagesCollection.EntityType.HasKey(pkg => pkg.Version);
+
             var downloadPackageAction = packagesCollection.EntityType.Action("Download");
-            downloadPackageAction.Returns<HttpResponseMessage>();
```

Comment:
We declare the "Download" method in the OData metadata document so that we can easily handle it in the OData service. The method happens to have a `HttpResponseMessage`. These are implementation details.

The problem with declaring the return type in the OData metadata document (which is what this line results in), is that `HttpResponseMessage` and all of its dependent types are then mapped to OData entities (e.g. the header collection). This is unnecessary and is noisy. Additionally, this is a delta from the latest NuGet.Server release and NuGet.org.